packages/abi-typegen/src/types/interfaces/JsonAbiNew.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Types for Fuel JSON ABI Format as defined on:
+ * https://github.com/FuelLabs/fuel-specs/blob/master/src/abi/json-abi-format.md
+ */
+export interface JsonAbi {
+  readonly specVersion: string;
+  readonly encodingVersion: string;
+  readonly programType: string;
```

Comment:
It makes sense, but nothing uses this yet, and it goes a bit beyond the scope of this PR.

Ultimately, it will probably be a shared `enum ProgramType` between all-things ABI.